crates/vault-registry/src/tests.rs

```diff
@@ -1036,40 +1041,43 @@ mod get_vaults_below_premium_collaterlization_tests {
             add_vault(id1.clone(), issue_tokens1, collateral1);
             add_vault(id2.clone(), issue_tokens2, collateral2);
 
+            // set back to default threshold so that vaults fall under premium redeem
+            set_default_thresholds();
+
             assert_eq!(
                 VaultRegistry::get_premium_redeem_vaults(),
-                Ok(vec![(id1, wrapped(issue_tokens1)), (id2, wrapped(issue_tokens2))])
+                Ok(vec![(id2, wrapped(52)), (id1, wrapped(51))])
             );
         })
     }
 
     #[test]
     fn get_vaults_below_premium_collateralization_filters_banned_and_sufficiently_collateralized_vaults() {
         run_test(|| {
-            // not returned, because is is not under premium threshold (which is set to 100% for this test)
+            // returned
             let id1 = vault_id(3);
             let issue_tokens1: u128 = 50;
             let collateral1 = 50;
             add_vault(id1.clone(), issue_tokens1, collateral1);
 
-            // returned
-            let id2 = vault_id(4);
-            let issue_tokens2: u128 = 50;
-            let collateral2 = 49;
-            add_vault(id2.clone(), issue_tokens2, collateral2);
-
```

Comment:
why did you modify this test?

Reply:
because the `max_premium`  calculation also considers `global_secure_threshold` which was not used by `redeemable_tokens` method 


- so start of the test set the secure collateral threshold to 0.001 so that the `add_vault`, method doesn't pass the `ExceedingVaultLimit`  error
- before calling the `get_premium_redeem_vaults` set thresholds back to default value so that `get_premium_redeem_vaults` works as expected.

Reply:
> because the `max_premium` calculation also considers `global_secure_threshold` which was not used by `redeemable_tokens` method earlier

This function is testing `get_premium_redeem_vaults` - that was also using `global_secure_threshold` before, right?

If premium threshold is set to 100% then vault1 should not be returned and   vault 2 should, that shouldn't have changed

Reply:
reverted back the test case, just have set back secure threshold after `add_vault` method since before `get_premium_redeem_vaults` just used `premium_threshold` now the `max_premium` calculation also uses `global_secure_threshold`.